### PR TITLE
fix: Strategies changes

### PR DESCRIPTION
**Pull request by samuveth**
### Issues

Fixes https://github.com/snapshot-labs/snapshot/issues/3778

### Changes

1. Remove the `Strategies` page in settings and move the `Voting strategies` block into `Voting` page image

3. Add link to strategie item name that links to the information page
4. Fix text

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [ ] I have tested my changes on a custom domain
- [ ] I have run end-to-end tests `yarn cypress:test:e2e`, and they have passed

### Additional notes
Including a "Strategies" page within the settings implies that strategies function as universal settings. However, these strategies are solely applicable to the voting process. Relocating them to the voting page would provide greater clarity on their purpose.

### CODE REVIEW

File: src/components/StrategiesListItem.vue [hunk 1]
```diff
@@ -18,7 +18,12 @@ defineEmits(['delete', 'edit']);
 <template>
   <BaseBlock slim class="group mb-3 p-4 text-skin-link">
     <div class="items-center justify-between sm:flex">
-      <h3 class="my-0 leading-5" v-text="strategy.name" />
+      <BaseLink
+        hide-external-icon
+        :link="`https://snapshot.org/#/strategy/${strategy.name}`"
```
Comment (bonustrack) [inline src/components/StrategiesListItem.vue:23]: This will not work on demo site

**Comment by bonustrack** [comment]
How does it help the issue to merge voting strategies into "voting" section?

**Comment by bonustrack** [comment]
Not sure the main link (strategy title) is the best to link to the strategy page, I believe main link should be edit the strategy params

**Comment by samuveth** [comment]
> How does it help the issue to merge voting strategies into "voting" section?

There is no issue for it, and the title of the PR suggests that there are multiple changes.

File: src/components/StrategiesListItem.vue (hunk 1 above)
Comment (samuveth) [reply src/components/StrategiesListItem.vue:23]: Why not?

Comment (samuveth) [reply src/components/StrategiesListItem.vue:23]: Playground is the same on demo and prod

**Comment by samuveth** [comment]
> Not sure the main link (strategy title) is the best to link to the strategy page, I believe main link should be edit the strategy params

Not a fan of the main link to info either tbh. We could just add another icon like this 
image

File: src/components/StrategiesListItem.vue (hunk 1 above)
Comment (samuveth) [reply src/components/StrategiesListItem.vue:23]: It will open in a new tab

Comment (bonustrack) [reply src/components/StrategiesListItem.vue:23]: But user will think he is on the demo site which add some confusion

**Comment by samuveth** [comment]
Okay, will create another PR for it to keep things separate

**Pull request closed without merging**